umm-lib/src/cmr/umm/iso_smap/granule/orbit.clj

```diff
@@ -50,40 +51,95 @@
             start-lat (convert-direction start-direction)
             end-lat (convert-direction end-direction))))
 
+(defn- build-ocsd-string
+  "Builds ISO SMAP orbit calculated spatial domain string using umm values."
+  [ocsd]
+  (let [{:keys [orbital-model-name orbit-number start-orbit-number stop-orbit-number
+                equator-crossing-longitude equator-crossing-date-time]} ocsd
+        OrbitalModelName (when orbital-model-name
+                           (str "OrbitalModelName: " orbital-model-name))
+        OrbitNumber (when orbit-number
+                      (str "OrbitNumber: " orbit-number))
+        StartOrbitNumber (when start-orbit-number
+                           (str "BeginOrbitNumber: " start-orbit-number))
+        StopOrbitNumber (when stop-orbit-number
+                          (str "EndOrbitNumber: " stop-orbit-number))
+        EquatorCrossingLongitude (when equator-crossing-longitude
+                                   (str "EquatorCrossingLongitude: " equator-crossing-longitude))
+        EquatorCrossingDateTime (when equator-crossing-date-time
+                                  (str "EquatorCrossingDateTime: " equator-crossing-date-time))]
+    (str OrbitalModelName OrbitNumber StartOrbitNumber StopOrbitNumber
+         EquatorCrossingLongitude EquatorCrossingDateTime)))
+
 (defn- parse-float
   "Coerce's string to float, catches exceptions and logs error message and returns nil if
   value is not parseable."
   [field value]
   (try
     (Float. value)
     (catch Exception e
-      (info (format "For orbit field [%s] the value [%s] is not a number." field value))
+      (info (format "For orbit/orbit-cal-spatial field [%s] the value [%s] is not a number." field value))
       ;; We return nil her instead of value because within-range validation can't handle comparing a
       ;; string to a double.
       nil)))
 
+(defn- parse-integer
+  "Coerce's string to integer, catches exceptions and logs error message and returns nil if
+  value is not parseable."
+  [field value]
+  (try
+    (Integer. value)
+    (catch Exception e
+      (info (format "For orbit-cal-spatial field [%s] the value [%s] is not an integer." field value))
```

Comment:
Make the error message consistent with what is in `parse-float`, i.e. use either `orbit/orbit-cal-spatial` or `orbit-cal-spatial`.

Reply:
Changed it to use "orbit" for both parse-float and parse-integer, since orbit-cal-spatial could be part of "orbit" as a generic term.